Essentials/src/messages.properties

```diff
@@ -475,8 +477,8 @@ skullChanged=\u00a76Skull changed to \u00a7c{0}\u00a76.
 slimeMalformedSize=\u00a74Malformed size.
 socialSpy=\u00a76SocialSpy for \u00a7c{0}\u00a76\: \u00a7c{1}
 socialSpyMsgFormat=\u00a76[\u00a7c{0}\u00a76 -> \u00a7c{1}\u00a76] \u00a77{2}
-socialSpyMutedPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a77(muted) \u00a7r
-socialSpyPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a7r
+socialSpyMutedPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a77(muted) \u00a7r{0}: {1}
+socialSpyPrefix=\u00a7f[\u00a76SS\u00a7f] \u00a7r{0}: {1}
```

Comment:
I'm not sure about this, the `Prefix` is supposed to go at the start. Also, if this is changed then suddenly everyone's custom messages will break.

Maybe rename these to `socialSpyMuted` and `socialSpy`?

Reply:
Oh, right... I did not take it. This is not good, I'll restore it.